November news (#675)
ropensci/roweb3: content/blog/2023-11-24-news-november-2023/index.Rmd

slug: news-november-2023
description: Donate to rOpenSci, Web assembly and R-universe, package news and new packages
params:
  last_newsletter: "2023-10-20"
---

```{r setup, include=FALSE}
library("magrittr")
library("rlang")
last_newsletter <- anytime::anytime(params$last_newsletter)
knitr::opts_chunk$set(echo = FALSE)
url <- sprintf(
    "/blog/%s/%s/%s/%s",
    lubridate::year(rmarkdown::yaml_front_matter(knitr::current_input())$date),
    stringr::str_pad(lubridate::month(rmarkdown::yaml_front_matter(knitr::current_input())$date), 2, "0", side = "left"),
    stringr::str_pad(lubridate::day(rmarkdown::yaml_front_matter(knitr::current_input())$date), 2, "0", side = "left"),
    rmarkdown::yaml_front_matter(knitr::current_input())$slug
    )
english <- function(x) {
  as.character(english::english(x))
}

nice_string <- function(...) {
  glue::glue_collapse(..., sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### Giving season

> PLACEHOLDER FOR SUMMARY OF YANI'S POST

Reviewer: ```suggestion

This Giving Season consider donating to rOpenSci to support our mission of empowering Open Science.

Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.

By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 

Join us in this vital mission today! You can donate here: https://ropensci.org/donate/
```

### Web assembly and R-universe

PLACEHOLDER, JEROEN TO WRITE A TECH NOTE

> ### Coworking

Author: please add it!

Read [all about coworking](/blog/2023/06/21/coworking/) in our recent [post](/blog/2023/06/21/coworking/)!

Join us for social coworking & office hours monthly on first Tuesdays! 
Hosted by Steffi LaZerte and various community hosts. 
Everyone welcome. 
No RSVP needed. 
Consult our [Events](/events) page to find your local time and how to join.

- [Tuesday, December 5th, 9:00 Australian Western (01:00 UTC)](/events/coworking-2023-12/), Why won't it pass?! Troubleshooting R package checks. With community host [Adam Sparks](/author/adam-sparks/) and [Steffi LaZerte](/author/steffi-lazerte/).
    - Explore what R package checks are and where they might fail;

>     - Work on an R package and run some checks;
>     - Spend some time troubleshooting your own failing package checks.

Reviewer 2: ```suggestion
    - Spend some time troubleshooting your own failing package checks;
    - Discuss tips and tricks for troubleshooting package checks with our community host and other attendees.
```

> 

Reviewer 2: ```suggestion

- [Tuesday, January 9th, 14:00 Europe Central (13:00 UTC)](/events/coworking-2024-01/), Working with APIs. With community host [Jon Harmon](/author/jon-harmon/) and [Steffi LaZerte](/author/steffi-lazerte/).
    - Explore using APIs in your own work;
    - Learn about packages that can help;
    - Discuss using APIs in R with our community host and other attendees.
    
```

If you're interested in maintaining any of the R packages below, you might enjoy reading our blog post [What Does It Mean to Maintain a Package?](/blog/2023/02/07/what-does-it-mean-to-maintain-a-package/) (or listening to its discussion on the [R Weekly highlights podcast](https://rweekly.fireside.fm/111) hosted by Eric Nantz and Mike Thomas)!

- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<https://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).

### Call for co-maintainers

Refer to our somewhat [recent blog post](/blog/2022/10/17/maintain-or-co-maintain-an-ropensci-package/#packages-looking-for-co-maintainers) to identify other packages where help is especially wished for!
See also our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.

> ## Package development corner

Author: :smile_cat: thank you!
